docs/source/tutorials/filtsmooth/linear_bayesian_filtering_smoothing.ipynb

```diff
@@ -0,0 +1,4552 @@
+{
```

Comment:
<p>Applied Stochastic Differential Equations</p><p>Bayesian Filtering and Smoothing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>